test/functional/p2p_invalid_block.py

```diff
@@ -78,9 +78,19 @@ def run_test(self):
         block2.vtx.append(tx2)
         assert_equal(block2.hashMerkleRoot, block2.calc_merkle_root())
         assert_equal(orig_hash, block2.rehash())
-        assert(block2_orig.vtx != block2.vtx)
+        assert block2_orig.vtx != block2.vtx
 
-        node.p2p.send_blocks_and_test([block2], node, success=False, request_block=False, reject_code=16, reject_reason=b'bad-txns-duplicate')
+        node.p2p.send_blocks_and_test([block2], node, success=False, reject_reason=b'bad-txns-duplicate')
```

Comment:
14700: original PR removed `request_block=False,` only, should keep `reject_code=16,` part